.gitlab-ci.yml

```diff
@@ -912,13 +912,19 @@ check_already_deployed_version:
 
 # deploy debian packages to apt staging repo
 deploy_deb:
-  <<: *run_when_triggered
   stage: deploy
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deploy:latest
   before_script:
-    - ls $OMNIBUS_PACKAGE_DIR
+    - mkdir -p $OMNIBUS_PACKAGE_DIR
   tags: [ "runner:main", "size:large" ]
   script:
+    # We first check that the current version hasn't already been deployed
+    # (same as the check_already_deployed_version). We do this twice to mitigate
+    # races and issues with retries while failing early if there is an issue.
+    - pushd $OMNIBUS_PACKAGE_DIR
+    - touch datadog-agent_6.5.2-1_amd64.deb
```

Comment:
Why do we touch that file ?

Reply:
https://github.com/DataDog/datadog-agent/commit/0d006d0a22c76f15ba5cf40613d8ff04cbb5f182 was added for testing. I used `touch` to create a fake `deb` that should make the pipeline fail.
This commit will be reverted before merging

Reply:
:+1: